mlflow/transformers/__init__.py

```diff
@@ -94,6 +100,10 @@
 _IMAGE_PROCESSOR_TYPE_KEY = "image_processor_type"
 _INFERENCE_CONFIG_BINARY_KEY = "inference_config.txt"
 _INSTANCE_TYPE_KEY = "instance_type"
+_INFERENCE_TASK_KEY = "inference_task"
```

Comment:
nit: Could you move these new constants incl. supported pipeline types to `inference_utils.py`, for better segregation?